planner/core/testdata/integration_suite_out.json

```diff
@@ -2347,10 +2347,10 @@
       {
         "SQL": "explain format = 'verbose' select count(*) from t2",
         "Plan": [
-          "StreamAgg_25 1.00 8.18 root  funcs:count(Column#7)->Column#4",
-          "└─TableReader_26 1.00 5.17 root  data:StreamAgg_9",
-          "  └─StreamAgg_9 1.00 49.50 batchCop[tiflash]  funcs:count(1)->Column#7",
-          "    └─TableFullScan_24 3.00 40.50 batchCop[tiflash] table:t2 keep order:false"
+          "StreamAgg_25 1.00 10.57 root  funcs:count(Column#7)->Column#4",
+          "└─TableReader_26 1.00 7.58 root  data:StreamAgg_9",
+          "  └─StreamAgg_9 1.00 85.50 batchCop[tiflash]  funcs:count(1)->Column#7",
+          "    └─TableFullScan_24 3.00 76.50 batchCop[tiflash] table:t2 keep order:false"
```

Comment:
Why is this cost changed?

Reply:
It is caused by `ts.HandleCols` is not set correctly, fixed.